target/linux/ath79/base-files/etc/hotplug.d/ieee80211/10_fix_wifi_mac

```diff
@@ -13,6 +13,11 @@ PHYNBR=${DEVPATH##*/phy}
 board=$(board_name)
 
 case "$board" in
+	adtran,bsap1800v2|\
+	adtran,bsap1840)
+		base_mac=$(mtd_get_mac_binary "Board data" 2)
+		echo $(macaddr_add "$base_mac" $(($PHYNBR * 8 + 1))) > /sys${DEVPATH}/macaddress
```

Comment:
Hi, useless echo. Replace with:
`macaddr_add "$base_mac" $(($PHYNBR * 8 + 1)) > /sys${DEVPATH}/macaddress`

Reply:
Oops. You're right.

Reply:
Wait, why is the rest of the file like that too?

Reply:
I don't know. I have sent a patch for that earlier today:
https://patchwork.ozlabs.org/patch/1120551/

Reply:
Patch has been accepted: https://github.com/openwrt/openwrt/commit/b16faf9bc725134c54a5976d979e94c9feb75446

Reply:
I'm still waiting for @ynezz to respond to the other comments before I push an update.

Reply:
Sorry, but I don't see any pending comments from you. ar71xx is simply gone, don't bother with that.